src/plugins/embeddable/public/lib/embeddables/embeddable_factory.ts

```diff
@@ -44,7 +45,8 @@ export interface EmbeddableFactory<
     TEmbeddableOutput
   >,
   TSavedObjectAttributes extends SavedObjectAttributes = SavedObjectAttributes
-> {
+  // @ts-ignore
```

Comment:
This is worrying in such a basic type - why do you need to shut up TS like this?